src/sidebar/components/Annotation/AnnotationShareControl.tsx

```diff
@@ -151,14 +151,14 @@ function AnnotationShareControl({
             )}
             width="custom"
           >
-            <h2 className="text-brand text-lg font-medium">
+            <h2 className="text-brand text-md font-medium">
```

Comment:
You'll see this throughout: Every type size is "shifted down one level." But the net result is that everything looks the same as it did before.